tests/test_slogpy_console.py

```diff
@@ -0,0 +1,80 @@
+"""The purpose of the code is to give all methods a unit test to make sure that they are working properly"""
```

Comment:
Update to use the proper way to set comments in python https://www.w3schools.com/python/python_comments.asp

Same for all other triple quoted strings meant to be comments below

Reply:
Actually, this is a docstring and triple quotes is correct.